src/schema/nmdc.yaml

```diff
@@ -2873,3 +2874,16 @@ slots:
     contributors:
       - ORCID:0009-0001-1555-1601 #Anastasiya Prymolenna
       - ORCID:0000-0002-8683-0050 #Montana Smith
+
+  bulk_elect_conductivity:
+    comments:
+      - Compatible with EMSL metadata
+    description: 
+      Electrical conductivity is a measure of the bulk soil ability to carry electric 
+      current, which is mostly dictated by the chemistry of and amount of soil water. 
+      Provide the value output of the field instrument.
+    title: bulk electrical conductivity
+    examples:
+      - value: 0.017 mS/cm
+    is_a: attribute
```

Comment:
i think that we have been inconsistent in saying that `Biosample` slots are `attribute`s. THis is fine, but we should start a new issue do do it more consistently.

Reply:
I removed "is_a: attribute" since it doesn't seem to be used consistently